src/main/java/org/broadinstitute/hellbender/utils/genotyper/ReadLikelihoods.java

```diff
@@ -1339,13 +1300,11 @@ private void removeSampleReads(final int sampleIndex, final Collection<GATKRead>
     public PerReadAlleleLikelihoodMap toPerReadAlleleLikelihoodMap(final int sampleIndex) {
         Utils.validIndex(sampleIndex, samples.numberOfSamples());
         final PerReadAlleleLikelihoodMap result = new PerReadAlleleLikelihoodMap();
-        final int alleleCount = alleles.numberOfAlleles();
         final GATKRead[] sampleReads = readsBySampleIndex[sampleIndex];
-        final int sampleReadCount = sampleReads.length;
-        for (int a = 0; a < alleleCount; a++) {
+        for (int a = 0; a < alleles.numberOfAlleles(); a++) {
```

Comment:
perf comment: this will call `alleles.numberOfAlleles()` on every iteration of the loop. Previously, it was once, before the loop. 

Normally I would not care but these methods may be on the critical path in HC or GenotypeGVCFs. Can you run the HC on your branch to make sure performance is not degraded